Header header
sensor_msgs/Image left
sensor_msgs/Image center
sensor_msgs/Image right